docs/marionette.application.md

```diff
@@ -160,6 +160,21 @@ allows you to provide extra configuration for various parts of your app througho
 initialization sequence.
 
 ```javascript
+var App = Mn.Application.extend({
```

Comment:
Made some really similar example https://jsfiddle.net/marionettejs/k05dctyt/ to one we have above.